addons/event/models/event_config_settings.py

```diff
@@ -31,9 +34,20 @@ class event_config_settings(osv.TransientModel):
             (1, 'Schedule emails to attendees and subscribers')
             ], "Email Scheduling",
             help='You will be able to configure emails, and to schedule them to be automatically sent to the attendees on subscription and/or attendance',
-            implied_group='event.group_email_scheduling'),            
+            implied_group='event.group_email_scheduling'),
     }
 
+    def onchange_custom_install_module(self, cr, uid, ids, custom_install_module, context=None):
+        if custom_install_module == 3:
+            return {'value': {'module_website_event_sale': True}}
+        elif custom_install_module == 2:
+            return {'value': {'module_event_sale': True, 'module_website_event_sale': False}}
+        return {'value': {'module_event_sale': False, 'module_website_event_sale': False}}
+
+    def set_default_custom_install_module(self, cr, uid, ids, context=None):
+        config_value = self.browse(cr, uid, ids, context=context).custom_install_module
+        self.pool['ir.values'].set_default(cr, uid, 'event.config.settings', 'custom_install_module', config_value)
```

Comment:
why need to store custom_install_module value in ir.values ?

I think, value of custom_install_module field should be set base on value of module_event_sale, module_website_event_sale.